### ISSUE TITLE
List of issues I am noticing (updated as I see more)

### ISSUE THREAD

**Issue by gr8pefish** [Carbon-Quadoxide-Initiative/Cub#2]
1) HQM mode was in edit (just remember to change that config setting before release)
2) Wasn't in adventure mode (another pre-release thing)
3) Unclear progression in beginning with map. The book didn't give hints as to the reason for there being a spawn/storage area and a tree-farm area.
4) Unclear why you spawn with machines (if that is intended). Also why you spawn with a written book and with a piece of stone.
5) The ore book you spawn with is useless. It just says "Found some ore today." On a related note, the quest that tells you to find the ore mining book is unclear/very hard (I have no idea where to look for it).
6) The fact that you are supposed to make a flint axe from the barrels near the tree farm area spawn is unclear, as is the fact that you have to use shears to get to the tree farm (because leaves are in the way). Also, might want to mention that you would need to get flint first in survival with your bare hands.
7) Might want to mention that tools have less durability and that items stack less.
8) When I try and claim a reward in hqm I get the reward but the book still says unclaimed reward. Not sure if that is on your guy's end or on vswe's.
9) Mining cobble takes forever (is it really supposed to be that slow?) If so, maybe give like 4 cobble as a previous quest reward so you only have to get a couple cobble for the furnace.
10) A bit of the hqm icons are showing as blank. Example: craft a flint pickaxe quest, they are all blank except the flint reward (unsure if that is just hqm misbehaving with rendering gt items or something else).
11) Bronze age quest treeline was unlocked right from the start (should be unlocked later).
12) Coal ore location totally unclear/unknown, so I had to cheat in some coal to complete that quest.
13) ToDo: Vein locations of tin/copper/and iron
.
.
.
More work needs to be done before it is tested thoroughly.

**Comment by mr10movie** [comment]
1. We know, we're mid-development so that would be the case :p
2. Same
3. That we do need to think about, thanks.
4. That's probably just the inventory we have on us, like I said, mid-development.
5. \+ 6. II'll look into it.
6. I thought we did no? Hmm...
7. Maybe if you're in creative mode.
8. Hard cobble might be on, I'll take a look.
9. That's a bug with GT and HQM, nothing we can do I'm afraid.
   11-13. Testing :P

I knew about most of these, but I probably should have made it clearer. I wanted you to look through the quests and see if  they explained themselves enough, or was there some information that the quest is leaving out that is vital for it's completion. 

Basically, are you learning anything from the pack (even in it's early state), or could you learn something if the pack was fully set up? Is the style of quest good? That's what I really want to know.

**Comment by gr8pefish** [comment]
Ah, sorry for the misunderstanding. 

Well everything I listed above that had 'unclear' in it's description needs refinement. Some specifics:

Ore generation is not explained at all, and while this map is more structured it would still be nice to know.
Basic steam/bronze progression seemed alright. It is hard to comment in depth as many pages said something like "Write description here" etc. I can't comment on the electric age, as I couldn't access that questline.

When you guys have done more work feel free to let me know again, but it really just isn't done enough for me to give you very helpful feedback. I'm not rushing you, I know how it goes ;P but it just isn't time yet.

**Comment by mr10movie** [comment]
Yeah, there was a reason we were looking for feedback for cat, and not cub XD. But you offered and the paragraph you just gave is actually vital for us to make progress. Thanks :)

**Comment by TheOnlyQueue** [comment]
Wow this was awesome feedback. Mr10movie is right, most of these are just because we are mid development, but nevertheless: thank you very much!

**Comment by TheOnlyQueue** [comment]
Let's keep this open for now

**Comment by CarbonBasedGhost** [comment]
Cat: We need to implement the Let's Greg ther mods scripts with minetweakr

**Comment by CarbonBasedGhost** [comment]
8) When I try and claim a reward in hqm I get the reward but the book still says unclaimed reward. Not sure if that is on your guy's end or on vswe's.

That is just because it is in edit mode :stuck_out_tongue:

**Comment by CarbonBasedGhost** [comment]
6) The fact that you are supposed to make a flint axe from the barrels near the tree farm area spawn is unclear, as is the fact that you have to use shears to get to the tree farm (because leaves are in the way). Also, might want to mention that you would need to get flint first in survival with your bare hands.

Added flint axe, we may need to explain a little bit more how adventure mode works

**Comment by TheOnlyQueue** [comment]
I'm free tomorrow, I'll hop on IRC then to discuss these and take a look at
those issues.

On Sun, Feb 1, 2015 at 2:59 PM, CarbonBasedGhost notifications@github.com
wrote:

> 6) The fact that you are supposed to make a flint axe from the barrels
> near the tree farm area spawn is unclear, as is the fact that you have to
> use shears to get to the tree farm (because leaves are in the way). Also,
> might want to mention that you would need to get flint first in survival
> with your bare hands.
> 
> Added flint axe, we may need to explain a little bit more how adventure
> mode works
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Carbon-Quadoxide-Initiative/Cub/issues/2#issuecomment-72381131
> .

## 

---

J Calkins

**Comment by gr8pefish** [comment]
If you guys need clarification on anything, just reply here. I am not playing the map, I am busy with other stuff (mod development being one thing), but I still thought I would offer to clear up anything I said in the original post if necessary.

**Comment by TheOnlyQueue** [comment]
Thank you, we will keep that in mind.
